src/Ledger/Foreign/HSLedger.agda

```diff
@@ -93,50 +93,57 @@ HSScriptStructure = record { p1s = HSP1ScriptStructure ; ps = HSP2ScriptStructur
 
 open import Data.Nat
 open import Data.Nat.Properties using (+-0-commutativeMonoid; _≟_)
-open import Ledger.TokenAlgebra
+open import Ledger.TokenAlgebra.Base using (TokenAlgebra)
```

Comment:
We should revert the reorg that moves TokenAlgebra definition to `Ledger.TokenAlgebra.Base`, if possible, or at least do it after holes in `TokenAlgebra/ValueSet.lagda` are all filled.